pyro/distributions/__init__.py

```diff
@@ -12,7 +8,25 @@
 from pyro.distributions.omt_mvn import OMTMultivariateNormal
 from pyro.distributions.rejector import Rejector
 from pyro.distributions.sparse_mvn import SparseMultivariateNormal
+from pyro.distributions.torch import *  # noqa F403
+from pyro.distributions.torch import __all__ as torch_dists
```

Comment:
isnt `__all__` more inclusive than `*`? im a bit confused what this and the previous line is doing

Reply:
The reason why you need the `import *` is to have the torch distributions in the local namespace. Just adding to the `__all__` isn't sufficient- it basically will add a string, but it has no information where to look for the class/function corresponding to the name.

Reply:
maybe i wasnt clear, i dont mean the `__all__ = ...` below, i mean between `from class import *` and `from class import __all__`, the latter should import everything the former does?